### PR TITLE
add new branch

### PR DESCRIPTION
**Pull request by PavloLevko**
I add new branch.  Also I add new class PathChach

### CODE REVIEW

File: src/DecryptSix.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+public class DecryptSix extends MyFileRider {
+    private static final String AFTER_DECRYPTED = "You text after decrypted.";
+    static char[] arrayAfterDecrypted = new char[100];
+    private static final int key = 1;
+
+    public static void decryptedToSix(char[] array) {
+
+        for (int i = 0; i < array.length; i++) {
+            char charFromArray = array[i];
+            int codeChar = charFromArray - 6;
```
Comment (Volodymyr95) [inline src/DecryptSix.java]: 6 - is magic number

File: src/DecryptSix.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+public class DecryptSix extends MyFileRider {
+    private static final String AFTER_DECRYPTED = "You text after decrypted.";
+    static char[] arrayAfterDecrypted = new char[100];
```
Comment (Volodymyr95) [inline src/DecryptSix.java]: 100 is magic number

File: src/DecryptThree.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    public static void decryptedToThree(char[] array) {
+
+        for (int i = 0; i < array.length; i++) {
+            char charFromArray = array[i];
+            int codeChar = charFromArray - 3;
+            arrayAfterDecrypted[i] = (char) codeChar;
+        }
+        System.out.println(arrayAfterDecrypted);
+        System.out.println(AFTER_ENCRYPTED);
+
```
Comment (Volodymyr95) [inline src/DecryptThree.java]: delete empty line

File: src/DecryptThree.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
+public class DecryptThree extends MyFileRider {
+    private static final String AFTER_ENCRYPTED = "You text after decrypted.";
+    static char[] arrayAfterDecrypted = new char[100];
+    private static final int key = 1;
+
+    public static void decryptedToThree(char[] array) {
+
+        for (int i = 0; i < array.length; i++) {
+            char charFromArray = array[i];
+            int codeChar = charFromArray - 3;
```
Comment (Volodymyr95) [inline src/DecryptThree.java]: 3 - is magic. number

File: src/Main.java [hunk 5]
```diff
@@ -1,7 +1,16 @@
...
-        UserMenu.sayHallo();
-        UserMenu.scanPath();
+    public static void main(String[] args) throws FileNotFoundException {
+       EncryptedToThree one = new EncryptedToThree();
+
+
+        UserMenu.mainMenu();
+
+
+
```
Comment (Volodymyr95) [inline src/Main.java]: delete empty lines

File: src/MyFileRider.java [hunk 6]
```diff
@@ -3,25 +3,24 @@
...
 import java.io.IOException;
 
-public class MyFileRider extends Main {
-
-
-    public static File file = new File("test.txt");
-    // Коли тут вводжу PATH одразу зявляється помилка NullPointException
 
+public class MyFileRider extends Main {
+    static String set;
```
Comment (Volodymyr95) [inline src/MyFileRider.java]: don't use static

File: src/PathCheck.java [hunk 7]
```diff
@@ -0,0 +1,22 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+
+
```
Comment (Volodymyr95) [inline src/PathCheck.java]: delete empty line and make formatting for all classes

File: src/PathCheck.java [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+public class PathCheck extends MyFileRider {
+
+    private static final String IS_EXISTS = "Correct path to file. Start reading...";
+    private static final String WRONG_PATH = "You input wrong path to file";
+    public static File file = new File("/Users/macbook/IdeaProjects/MyProject/src/test.txt");
```
Comment (Volodymyr95) [inline src/PathCheck.java]: hardcoded url, it shoud be passed from console(user input)

File: src/BruteForce.java [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+        if (valueSpaceThree>COEFFICIENT){
+            DecryptThree.decryptedToThree(MyFileRider.charsArrayAfterRead);
+        }else if(valueSpaceSix>COEFFICIENT) {
+            DecryptSix.decryptedToSix(MyFileRider.charsArrayAfterRead);
+        }else{
+            System.out.println(MASSAGE);
+        }
+    }
+
+
```
Comment (Volodymyr95) [inline src/BruteForce.java]: too mane empty lines, delete it

File: src/DecryptSix.java [hunk 10]
```diff
@@ -0,0 +1,18 @@
+public class DecryptSix extends MyFileRider {
+    private static final String AFTER_DECRYPTED = "You text after decrypted.";
+    private static final int numberSix = 6;
+    private static final int buferValue = 100;
+    static char[] arrayAfterDecrypted = new char[buferValue];
+    private static final int key = 1;
+    public static void decryptedToSix(char[] array) {
```
Comment (Volodymyr95) [inline src/DecryptSix.java]: what it's mean decryptedToSix?

File: src/DecryptThree.java [hunk 11]
```diff
@@ -0,0 +1,19 @@
+public class DecryptThree extends MyFileRider {
+    private static final String AFTER_ENCRYPTED = "You text after decrypted.";
+    private static final int numberThree = 3;
+    private static final int buferValue = 100;
+    static char[] arrayAfterDecrypted = new char[buferValue];
+    private static final int key = 1;
+    public static void decryptedToThree(char[] array) {
```
Comment (Volodymyr95) [inline src/DecryptThree.java]: what it's mean decryptedToThree?

File: src/Main.java [hunk 12]
```diff
@@ -1,8 +1,6 @@
+import java.io.FileNotFoundException;
 public class Main {
-    public static void main(String[] args) {
-        UserMenu.sayHallo();
-        UserMenu.scanPath();
+    public static void main(String[] args) throws FileNotFoundException {
+        UserMenu.mainMenu();
```
Comment (Volodymyr95) [inline src/Main.java]: it should not be a static

File: src/UserMenu.java [hunk 13]
```diff
@@ -1,41 +1,65 @@
...
+    private static final String START_METOD = "Please enter the file path.";
+    public static boolean isRaning = true;
+    public static String  pathScanner;
+        public static void mainMenu () throws FileNotFoundException {
+            System.out.println(MAIN_MENU);
+            Scanner scan = new Scanner(System.in);
+            while (isRaning) {
+                String inputText = scan.nextLine();
+                switch (inputText) {
+                    case "1" -> {
```
Comment (Volodymyr95) [inline src/UserMenu.java]: 1,2,3,4 - is a magic numbers

File: src/DecryptSix.java (hunk 10 above)
Comment (PavloLevko) [reply src/DecryptSix.java]: decrypted To Six  char

File: src/BruteForce.java [hunk 14]
```diff
@@ -0,0 +1,31 @@
+public class BruteForce {
+    private static final String MASSAGE = "Encryption does not math the conditions!";
+    private static final String BRUTE_FORCE_COMPLITE = "Brute Force is complite!";
+    private static final char SPACE_THREE= '#';
+    private static final char SPACE_SIX = '&';
+    public static int valueSpaceThree =0;
+    public static int valueSpaceSix =0;
+    private static final int COEFFICIENT = 3;
+    public static void bruteForse (char[] array) {
```
Comment (Volodymyr95) [inline src/BruteForce.java]: don't use static

File: src/EncryptedToSix.java [hunk 15]
```diff
@@ -0,0 +1,15 @@
+public class EncryptedToSix  {
+    private static final String AFTER_ENCRYPTED = "You text after encrypted.";
+    private static final int numberSix = 6;
```
Comment (Volodymyr95) [inline src/EncryptedToSix.java:3]: don't use static

File: src/BruteForce.java [hunk 16]
```diff
@@ -0,0 +1,40 @@
+public class BruteForce {
+    private static final String MASSAGE = "Encryption does not math the conditions!";
+    private static final String BRUTE_FORCE_COMPLITE = "Brute Force is complite!";
+    private static final char SPACE_THREE= '#';
+    private static final char SPACE_SIX = '&';
+    public static int getValueSpaceThree() {
```
Comment (Volodymyr95) [inline src/BruteForce.java:9]: + empty lines

File: src/BruteForce.java [hunk 17]
```diff
@@ -0,0 +1,40 @@
...
+        return valueSpaceThree;
+    }
+    public static int valueSpaceThree =0;
+
```
Comment (Volodymyr95) [inline src/BruteForce.java]: class order:
constants
object fields
constructors
public methods
private methods

File: src/UserMenu.java [hunk 18]
```diff
@@ -1,41 +1,74 @@
...
-                System.out.println(CORRECTPATH);
-
-            } else if (PATH.isEmpty()) {
-                System.out.println(CORRECTPATH);
-
-
-                isRaning = false;
-
-
+public class UserMenu  extends Main {
```
Comment (Volodymyr95) [inline src/UserMenu.java]: Menu, without extends

**Review by Volodymyr95 — APPROVED** [review]

**Pull request merged**